src/util/util_env.cpp

```diff
@@ -135,6 +135,10 @@ namespace dxvk::env {
         return getExecutableName() == std::string("aces.exe");
     }
 
+    bool isXDefiant() {
+        return getExecutableName() == std::string("upc.exe");
```

Comment:
`upc.exe` is Ubisoft Connect itself, game executable is called `XDefiant.exe` as far as I can tell.

Reply:
Whoops, already fixed, ignore me :sweat_smile: